lib/client/workloadendpoint_e2e_test.go

```diff
@@ -184,8 +164,8 @@ var _ = testutils.E2eDatastoreDescribe("WorkloadEndpoint tests", testutils.Datas
 					"prod": "no",
 				}},
 			api.WorkloadEndpointMetadata{
-				Name:         "ep1/with_foo",
```

Comment:
oh dammit - I actually copied this file across from my refactor branch.

I should really add that back in.  I will.